app/Kconfig

```diff
@@ -481,6 +494,21 @@ config ZMK_KSCAN_EVENT_QUEUE_SIZE
 
 endif # ZMK_KSCAN
 
+config ZMK_KSCAN_SIDEBAND_BEHAVIORS
+    bool
+    default y
+    depends on DT_HAS_ZMK_KSCAN_SIDEBAND_BEHAVIORS_ENABLED
+    select KSCAN
+
+if ZMK_KSCAN_SIDEBAND_BEHAVIORS
+
+config ZMK_KSCAN_SIDEBAND_BEHAVIORS_INIT_PRIORITY
+    int "Keyboard scan sideband behaviors driver init priority"
+    # The default kscan init priority is 90, so be sure we are lower.
```

Comment:
"be sure we are lower" paired with a value that is greater than 90 might be a bit confusing. Maybe say "initialized later" instead?

Reply:
Tweaked for clarity.